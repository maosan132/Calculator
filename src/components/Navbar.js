import React from 'react';
import { Link } from 'react-router-dom';

const NavBar = () => (
  <div id="navbar">
    <h1>Math Magicians</h1>
    <div className="nav">
      <Link to="/io/math-magicians/">Home </Link>
      <Link to="/io/math-magicians/App">Calculator </Link>
      <Link to="/io/math-magicians/Quote">Quote </Link>
    </div>
  </div>
);

export default NavBar;
