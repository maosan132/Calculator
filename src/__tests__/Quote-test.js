import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import Renderer from 'react-test-renderer';
import Navbar from '../components/Navbar';

describe('Quote', () => {
  it('renders correctly', () => {
    const quote = Renderer
      .create(
        <>
          <Router>
            <Navbar />
          </Router>
          <blockquote>Mathematics is about understanding.</blockquote>
        </>,
      )
      .toJSON();
    expect(quote).toMatchSnapshot();
  });
});
