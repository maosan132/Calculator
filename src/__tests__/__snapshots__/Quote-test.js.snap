// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote renders correctly 1`] = `
Array [
  <div
    id="navbar"
  >
    <h1>
      Math Magicians
    </h1>
    <div
      className="nav"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Home 
      </a>
      <a
        href="/App"
        onClick={[Function]}
      >
        Calculator 
      </a>
      <a
        href="/Quote"
        onClick={[Function]}
      >
        Quote 
      </a>
    </div>
  </div>,
  <blockquote>
    Mathematics is about understanding.
  </blockquote>,
]
`;
